tests/io/rbd_io.py

```diff
@@ -0,0 +1,78 @@
+import json
+import random
+import string
+
+from tests.io.io_utils import start_io
+from tests.rbd.rbd_utils import Rbd
+from utility.log import Log
+from utility.utils import convert_bytes, get_storage_stats, run_mkfs
+
+log = Log(__name__)
+
+
+class rbd_io:
+    """
+    This class is a tool for running different IOs on Ceph Block Device
+
+    Functionalities:
+    1. Creates the Rbd pool if not present
+    2. Creates the image with size of the data that needs to be filled
+    3. Mounts the image with extfs on the client nodes
+    4. Runs the SmallFile IOs on the mounted directory
```

Comment:
We already had  a module and method handling similar requirements
https://github.com/red-hat-storage/cephci/blob/master/tests/rbd/krbd_io_handler.py

Not sure whether the existing one could have been extended.

If we want to maintain two methods, we could add unmount, unmap options also to handle safe tear down options.

Reply:
Yes Vasi..we can use this.
I will make the changes to use this function

Reply:
Hi Vasi,
It has few taugtly coupled with only fio and we don;t have options to run other io tools
with the new class we can add as many methods and run IOs.
krbd also runs sequentially on each image. with the latest changes, we can run IOs using containers on multiple mount points.
Ultimately we are using same steps for mounting images 

Regards,
Amarnath